saleor/settings.py

```diff
@@ -194,24 +194,64 @@ def get_bool_from_env(name, default_value):
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = os.environ.get("SECRET_KEY")
 
-MIDDLEWARE = [
+# Only used to dispatch requests to correct middlewares, please don't directly use it.
+MIDDLEWARE = ["saleor.core.middleware.middleware_dispatcher"]
+
+# Middleware for Saleor 1.0 (storefront and dashboard)
+DJANGO_MIDDLEWARE = [
+    # SessionMiddleware stores and manages session data from/to a cookie.
     "django.contrib.sessions.middleware.SessionMiddleware",
+    # SecurityMiddleware redirect the users to HTTPS when required/requested.
     "django.middleware.security.SecurityMiddleware",
+    # CommonMiddleware has for job to:
```

Comment:
I think most of these comments are unnecessary, they're documenting parts of Django or third-party libraries and should be already documented their docs. When it comes to comments related to our middlewares, they should be added as docstrings inside middlewares' definitions.